sharding-repl-cache/scripts/mongo-init.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+
+echo "initialize config"
+docker compose exec -it configSvr mongosh --quiet <<EOF
```

Comment:
На линукс чтобы норм работало нужно так
```suggestion
docker compose exec -T configSvr mongosh --quiet <<EOF
```